client/src/Tickets/Tickets.jsx

```diff
@@ -103,15 +87,16 @@ const Tickets = () => {
         });
         setData(data);
       });
-    if (user) {
-      setTicket((prevTicket) => ({ ...prevTicket, UserId: user.id }));
-    }
-  }, [setData, user]);
+  }, []);
 
-  function updateTable(ticket) {
+  function onCreate(ticket) {
     setData([...data, ticket]);
   }
 
+  function onUpdate(ticket) {
+    setData(data.map((t) => (t.id == ticket.id ? { ...ticket } : t)));
```

Comment:
So we move this code back to this component, and not in the modal. The modal shouldn't have to know the internal data structure of the parent.